layersvt/VkLayer_device_simulation.json.in

```diff
@@ -6,6 +6,15 @@
         "library_path": "@RELATIVE_LAYER_BINARY@",
         "api_version": "@VK_VERSION@",
         "implementation_version": "1.2.6",
```

Comment:
could you edit this to the current "1.3.0" while you're in here?

Reply:
Done.